casbin/log/log.py

```diff
@@ -5,8 +5,7 @@
 
 def set_logger(l):
     """sets the current logger."""
-    logger = l
-
+    logger.logger =l
```

Comment:
set_logger should set the logger under the package, not the DefaultLogger.
This should refer to the Golang version of casbin.